alexnet/problem.py

```diff
@@ -5,54 +5,77 @@
 # sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 from torch_wrapper import use_knl  # noqa
 
-
+#NOTE: I have intentially left the ranges for some of the hyper-parameters large in order to facilitate a large number of combinations. There will be configurations of these ranges that cause errors related to kernels being larger than images, but I am choosing to ignore those cases for now because limiting the ranges would greatly reduce the valid combinations of hyper-parameters. Feel free to change ranges based on your needs.
```

Comment:
Can you wrap this (and any other long lines) at 88 characters?

https://github.com/felker/deephyper_pytorch_layers/blob/master/.pre-commit-config.yaml 

Should auto-format the code for you when you attempt to `git commit`, as long as `pip install pre-commit flake8 black`

Reply:
also I think it is nice to sign `# TODO(MS):` and `# NOTE(MS):` comments with the author's initials

Reply:
I have auto-formatted the code as per-instructions, however, I don't think comments are auto-formatted. I went ahead and manually formatted the comments to fit requirments.

Reply:
Yeah turns out comments and long strings wont automatically be fixed by `black`